#171784074 Configure ESLint

Francois-MUGOROZI opened the pull request:
#### What does this PR do?
 Having ESLint configured.

#### Description of Task to be completed?
  Having ESLint configured to help check for style violations and flag them.
  Having ESLint configured to use Airbnb convention

#### How should this be manually tested?
After cloning the repo, cd into it, create a <name>.js file
and then try different codes like importing module which does exist and so on.
You will notice warning or error.

#### What are the relevant pivotal tracker stories?
[#171784074](https://www.pivotaltracker.com/n/projects/2438874/stories/171784074)

irfiacre commented on package.json:
```diff
@@ -0,0 +1,31 @@
+    "eslint-plugin-prettier": "^3.1.2",
+    "prettier": "^2.0.1"
+  }
+}
```
please add an empty line

Francois-MUGOROZI committed:
chore(Eslint): add Eslint
   - added .eslintrc.json config file
   - added .prettierrc.json config
   The eslint is configured to use es6 and airbnb conventions

 [Finish #171784074]

fordarnold reviewed (commented):
A minor issue on my side... The `rules:{}` section seems to be having too many rules ignored. Won't this mess things up on during linting.

nkalyesubula reviewed (changes requested):
Well done Francios. A few change requests though,  Please add package-lock.json in .gitignore file, correct the PR title based on the format I shared with you and also make the story ID clickable i.e link it to the PT board

Francois-MUGOROZI committed:
chore(Eslint): add Eslint-correcting feedback

   -added package-lock to gitignore

 [Finish #171784074]
